src/core_atmosphere/Registry.xml

```diff
@@ -1983,45 +2105,46 @@
 
                         <var name="tend_qc" name_in_code="qc" array_group="moist" units="kg m^{-3} s^{-1}"
                              description="Tendency of cloud water mass per unit volume divided by d(zeta)/dz"
-                             packages="bl_mynn_in;bl_ysu_in;bl_myj_in;cu_ntiedtke_in;mp_kessler_in;mp_thompson_in;mp_wsm6_in;mp_nssl2m_in"/>
+                             packages="bl_mynn_in;bl_ysu_in;bl_myj_in;cu_nntiedtke_in;mp_kessler_in;mp_thompson_in;mp_thompson_aers_in;mp_wsm6_in;mp_nssl2m_in"/>
```

Comment:
This "cu_nntiedtke_in" error is happening so frequently, I'm starting to wonder if they actually changed the name of the scheme... Need to find out and do a regressive search and replace if necessary.

Reply:
They didn't change the name of the scheme, I think this is a result of auto-combining some of the conflicted lines